### PR TITLE
phase space mpl: fix update colorbar

### PR DESCRIPTION
**Pull request by kossag14**
rescaling of updated imshow data must update vmin, vmax

### CODE REVIEW

File: lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py [hunk 1]
```diff
@@ -54,13 +54,20 @@ def _create_plt_obj(self, ax):
...
+        ax.set_ylabel(r'$p_{0}$ [$\beta\gamma$]'.format(meta.p))
 
     def _update_plt_obj(self):
         """
         Implementation of base class function.
         """
         dat, meta = self.data
         self.plt_obj.set_data(np.abs(dat).T * meta.dV)
+        self.plt_obj.autoscale()
+        self.cbar.update_normal(self.plt_obj)
```
Comment (PrometheusPi) [inline lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py]: question:
does this line solve you rescaling issue?

Comment (ax3l) [reply lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py]: no, the line before updates the ranges and that one updates the cb drawing

Comment (PrometheusPi) [reply lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py]: thanks

File: lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py [hunk 2]
```diff
@@ -54,13 +54,20 @@ def _create_plt_obj(self, ax):
             origin='lower',
             norm=LogNorm()
         )
+        self.cbar = plt.colorbar(self.plt_obj, ax=ax)
+        self.cbar.set_label(
+                r'$Q / \mathrm{d}r \mathrm{d}p$ [$\mathrm{C s kg^{-1} m^{-2}}$] ')
```
[the remark this answers is not in the document]
Comment (ax3l) [reply lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py]: Also, it's over-intended (more than 4 spaces)

[the remark this answers is not in the document]
Comment (ax3l) [reply lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py]: You can run
```python
pip install -U flake8
flake8 lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py
```

to verify after modification

**Pull request merged**